### DISCUSSION TITLE
Serial connection parameters

### DISCUSSION THREAD

**Discussion by caillotantoine** [xjtuecho/EBAZ4205#23] [Q&A] [answered]
Hello, 
I am about to plug the board for the board for the first time and I was wondering about two questions about the serial connection: 
1. What are the voltage level? Is it 5V or 3V3?
2. What is the baud rate? Is it 115200 as we configure it when we reset the root password? 

Thank you for your answer and my apologies for such easy question.

**Comment by nic3-14159** [comment, marked as answer]
The serial connection is 3.3V and yes, the baud rate is 115200. However, that rate is simply what the original code on the NAND set, and can be changed if you ever build your own boot image to use with it.

**Reply by caillotantoine** [reply]
Awesome! Thank you for your quick answer!
